src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -1210,14 +1210,32 @@
 		private static MAX_WIDTH = `${Number.MAX_SAFE_INTEGER}vw`;
 		private static MAX_HEIGHT = `${Number.MAX_SAFE_INTEGER}vh`;
 
+		private measureElement(element: HTMLElement): [number, number] {
+
+			const offsetWidth = element.offsetWidth;
+			const offsetHeight = element.offsetHeight;
+
+			const resultWidth = offsetWidth ? offsetWidth : element.clientWidth;
+			const resultHeight = offsetHeight ? offsetHeight : element.clientHeight;
+
+			// +0.5 is added to take rounding into account
+			return [resultWidth + 0.5, resultHeight];
```

Comment:
+0.5 for the height too, no?

Reply:
Wasn't there before. We'll have to revisit that computation for sure.